### ISSUE TITLE
New replay format

### ISSUE THREAD

**Issue by WLTN1** [tpatel/fortnite-replay-reader#7]
Hello, epicgames changed replay format, will you updated lib please? Thank you so much

event PLAYER_ELIMINATION

**Comment by jokoast** [comment]
It seems that they replaced the player name by the unique player id. 

EDIT : 
The name is replaced by the epic account identifier, 100% sure. I think the Fortnite client resolves the EpicID to get the account name.

**Comment by WLTN1** [comment]
Yes, but its a update in offsets of parser and parser is now broken until he will be updated

**Comment by XeraFiu-YTB** [comment]
Hello, it's the same for me, i have this error :
RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 124. Received 3215298567
It's seems that the error comes at the condition ligne 206 to 210 of the parser.js, the Buffer is too long with 128 bytes..

Thx for any answers :octopus:

**Comment by harshitvyas98** [comment]
hey, will this parse unreal engine .replay file to JSON ??
